contexts/security-v3-unstable.jsonld

```diff
@@ -5,11 +5,15 @@
         },
         "https://w3id.org/security/v2",
         {
-            "ethereumAddress" {
+            "owl": "https://www.w3.org/TR/owl-ref/#",
```

Comment:
```suggestion
            "owl": "http://www.w3.org/2002/07/owl#",
```

Reply:
Yes, `http://www.w3.org/2002/07/owl#` is indeed the right OWL URL.